package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -512,7 +526,8 @@ def __init__(self, filename, lammps_coordinate_convention="auto",
     def _reopen(self):
         self.close()
         self._file = util.anyopen(self.filename)
-        self.ts = self._Timestep(self.n_atoms, **self._ts_kwargs)
+        self.ts = self._Timestep(self.n_atoms,
```

Comment:
Why is this being changed? It doesn't violate pep8 nor does it seem to have any functional changes?